lib/exercises.rb

```diff
@@ -1,29 +1,144 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(2nm + p) where n is the length of the list passed to grouped_anagrams, and m represents the number of chars in each string and p is the number of unique char hashes
+#                 == O(nm)
+# Space Complexity: Um O(2n) == O(n) since I am creating an array equal to the length of the passed in list and the worst case size for the meta_hash is also the length of the passed in list?
 
 def grouped_anagrams(strings)
```

Comment:
I'd say this is O(n * m log m) where m is the number of chars in each string and n is the length of the array.